src/lib/cards/CompletionCard.svelte

```diff
@@ -1,6 +1,6 @@
 <script lang="ts">
 	import type { Completion } from '$lib/types';
-	import { formatTime, getPersonColor } from '$lib/util';
+	import { formatTime, getPersonColor, linkIsLogfile, linkIsVideo } from '$lib/util';
```

Comment:
Is `linkIsLogfile` or `linkIsVideo` going to be used in other places? If not, it would be better to keep them in this file.

Reply:
Most probably we would add it to the Verify page, and/or the upload page when uploading.

Reply:
You are correct. I did not look into it that far.